pkg/segment/query/querystatus.go

```diff
@@ -925,3 +937,43 @@ func LogGlobalSearchErrors(qid uint64) {
 		putils.LogUsingLevel(errInfo.LogLevel, "qid=%v, %v, Count: %v, ExtraInfo: %v", qid, errMsg, errInfo.Count, errInfo.Error)
 	}
 }
+
+func SetNewQueryPipelineResponse(response *structs.PipeSearchResponseOuter, qid uint64) error {
```

Comment:
same comment as above, rename this func based on what name you give for the var above

Reply:
addressed in the latest commit.